### PR TITLE
Updated Launcher command in Readme

### PR DESCRIPTION
**Pull request by jonnyshaw89**
- Added new var to launcher in compose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ pipenv run ./scripts/run_app.sh
 Note, you will also need to run an upstream tool (eg, https://github.com/ONSDigital/go-launch-a-survey) to launch a survey.
 
 ```
-docker run -it -p 8000:8000 onsdigital/go-launch-a-survey:latest
+docker run -e SURVEY_RUNNER_SCHEMA_URL=http://docker.for.mac.host.internal:5000 -it -p 8000:8000 onsdigital/go-launch-a-survey:read-metadata-from-schema
```
Comment (dwyeradam) [inline README.md]: Just remember to revert `read-metadata-from-schema` to `latest`

**Comment by benhowes** [comment]
If you want to test this, you also need to change the docker compose to use the `read-metadata-from-schema` branch of the survey launcher. i.e.

```
go-launch-a-survey:
    image: onsdigital/go-launch-a-survey:read-metadata-from-schema
```

File: docker-compose.yml [hunk 2]
```diff
@@ -73,6 +73,7 @@ services:
     image: onsdigital/go-launch-a-survey:latest
```
Comment (dwyeradam) [inline docker-compose.yml:73]: I changed `latest` to `read-metadata-from-schema` for testing.

**Review by dwyeradam — APPROVED** [review]

**Review by benhowes — APPROVED** [review]
Was going to say the same things as @dwyeradam

**Pull request merged**